engine/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -169,10 +168,21 @@ LABEL com.example.version="0.0.1-beta"
 LABEL vendor="ACME Incorporated"
 LABEL com.example.release-date="2015-02-12"
 LABEL com.example.version.is-production=""
+```
+
+An image can have more than one label. To specify multiple labels, Docker
+recommends combining labels into a single `LABEL` instruction where possible.
+Each `LABEL` instruction produces a new layer which can result in an inefficient
+image if you use many labels. This example results in a single image layer.
```

Comment:
If I'm not mistaken, anything other than RUN, COPY, or ADD (i.e. commands that touch the file system) no longer create a layer; they only create an intermediate image during build (basically, just the changes in image configuration), but that doesn't affect the final image, nor it's size

Reply:
@thaJeztah Okay if this is true, then I'll need to either remove or re-word this here, and then also get the reference page updated (which does not live in the docs repository).

@mstanleyjones can you help here?